src/components/vouchers/Discount.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback, useContext } from 'react';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { BookmarkContext } from '../../BookmarkProvider';
+import { Icon, colors, consts, normalize } from '../../config';
+import { useBookmarkedStatus } from '../../hooks';
+import { togglePushDeviceAssignment } from '../../pushNotifications';
+import { TDiscount } from '../../types';
+import { BoldText, RegularText } from '../Text';
+
+const a11yLabel = consts.a11yLabel;
+
+export const Discount = ({
+  discount,
+  query,
+  id
+}: {
+  discount: TDiscount;
+  query: string;
+  id: string;
+}) => {
+  const { toggleBookmark } = useContext(BookmarkContext);
+  const { discountAmount, discountedPrice, originalPrice } = discount;
+
+  const isBookmarked = useBookmarkedStatus(query, id);
+
+  const onPress = useCallback(() => {
+    toggleBookmark(query, id);
+    togglePushDeviceAssignment(
+      id,
+      query.charAt(0).toUpperCase() + query.slice(1), // convert first character to uppercase
+      isBookmarked ? 'DELETE' : 'POST'
+    );
+  }, [id, query]);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.container}>
+        <View style={styles.discountedPriceContainer}>
+          <BoldText lightest>{discountedPrice} €</BoldText>
+        </View>
+
+        <RegularText lineThrough lighter>
+          {originalPrice} €
+        </RegularText>
+        <RegularText primary> - {discountAmount} €</RegularText>
```

Comment:
there shouldn't be a space between - and the discount amount

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/988/commits/de161547c3488ae2a87cbe6a5163c6cea10564e2